textBasedAdventure.js

```diff
@@ -1,5 +1,146 @@
+let spiritWraith = 25 // Ravenous shadows of the night - Feed and Kill Settler Population
+
+//Settler Power Counter
+let settlerCapitalPower = 50
+let miningComplexPower = 30
+let frontierGPower = 40
+let damPower = 30
+let farmsPower = 20
+
+const energyUsage = () => {
+    spiritEnergy += spiritPower[targetPower].Energy
+}
+
+const guardianPower = () => {
+    spiritPower = [{power: "Ignite Hellfire ", Energy: "15 Prisms"}, {power: "Starve the Settler Progeny ", Energy: 20}, {power: "Infect the Settlers with a Mortal Plague ", Energy: 20}, {power: "Disrupt Settler Communication", Energy: 20}, {power: "Incite Mass Purge Psychosis", Energy: 25}, {power: "Unleash the Wraith", Energy: 25}] 
+    targetPower = readline.keyInSelect(spiritPower, `Choose the Fate of the Outpost. Reckon with the sacrifice`,
+    {cancel: "The Coward's Way"});
+    console.log(`You have chosen to ${spiritPower[targetPower].power}. You have righteously sacrificed ${spiritPower[targetPower].Energy} Prisms from your energy nexus`)
+    energyUsage()
+}
+
+// const settlerDamage = () => {
+//     if (spiritPower[targetPower].power = spiritFire){
+//         if (colonialOutpost[targetOutpost] === colonialOutpost[0])
+
+//     }
+// }
+
+const attackSettler = () => {
+    if (colonialOutpost[targetOutpost] === colonialOutpost[0]){
+        guardianPower()
+        settlerDamage()
+    } else if (colonialOutpost[targetOutpost] === colonialOutpost[1]){
+        guardianPower()
+        settlerDamage()
+    }
+}
+
+
+const spiritDamage = () => { 
+    if (attack === settlerCapitalPower){
+    spiritHealth += settlerCapitalPower
+    } 
+}   
+
+
+//GAME SET-UP
+///////////////////////////////////////////////////////////////////////////////////////////////
+
+const outposts1 = () => {
+    colonialOutpost = ["The Settler Capital on the river delta ", "The Lead Ore Mining Complex in the hills of the Progenitors ", "The Freeholder Farms on the great pampas ", "The Frontier Garrison on the river terraces ", "TheWall damning our sacred waters "]
+    targetOutpost = readline.keyInSelect(colonialOutpost,`Which defiled outpost shall we attack ${spiritName}?`);
+    console.log(`Sunder ${colonialOutpost[targetOutpost]} `);
+    attackSettler()
+}
+outposts1()
+
+// const outposts = () => {
+//     colonialOutposts = ["The Settler Capital ", "The lead ore mining complex ", "The freeholder farms ", "The frontier garrison ", "The river wall "]
+//     targetOutpost = readline.keyInSelect(colonialOutposts, `Which defiled outpost shall we attack next? ${spiritName}`);
+//     console.log(` ${colonialOutposts[targetOutpost]} `);
+//     }
+
+
+    function nbYear(p0, percent, aug, p) {
+        let year = 1
+        for (let i = 0; i <= p; i++) {
+            let popInc = p0 + (p0 * (percent/100)) + aug
+            if (popInc < p) {
+                year++
+                p0 = popInc
+            }  
+        } 
+        return year
+    } 
```

Comment:
I think I know this one! ;) Make sure you remove any code that is not linked to the project.